scripts/validateTests.ts

```diff
@@ -0,0 +1,59 @@
+import * as fs from 'fs';
+import { IAssignmentTestCase } from './types';
+
+
+const parseJSON = (testFilePath: string) => {
+  try {
+    const fileContents = fs.readFileSync(testFilePath, 'utf-8');
+    const parsedJSON = JSON.parse(fileContents);
+    return parsedJSON as IAssignmentTestCase;
+  } catch (err) {
+    console.error(`failed to parse JSON in ${testFilePath}`)
+    console.error(err);
+    process.exit(1);
+  }
+}
+
+const validateAssignmentDetails = (testFilePath: string, testCase: IAssignmentTestCase) => {
```

Comment:
you could also turn this into a jest test and just run it with jest, might be easier to expand tests in the future that way

Reply:
updated!